plugins/out_file/file.c

```diff
@@ -76,12 +79,21 @@ static char *check_delimiter(const char *str)
     return NULL;
 }
 
+static int check_integer(const char *str)
+{
+    if (str == NULL) {
+        return 0;
+    }
+    return(atoi(str));
```

Comment:
I think this probably needs a lot more defensive code around it to handle all the possible failure outcomes and "undefined behaviour" of `atoi`: https://cplusplus.com/reference/cstdlib/atoi/

It should also test invalid strings, etc. to confirm.